docs/source/developer-guides/prs-and-reviews.rst

```diff
@@ -0,0 +1,75 @@
+Consolidation:
+==============
+
+Overarching Goals of good Review
+--------------------------------
+
+The goal of a review is to ensure that the proposed change is useful and maintainable over the long run.
+
+PR Description
+---------------
+
+- Have a meaningful title: it's easier when creating the changelog or when searching through old PR
+- Motivate why we should merge the PR: adding/changing code risks introducing a new bug. IMO one person asking for a nice to have feature doesn't qualify.
+- Explain the PR does in the description: it makes it easier/faster to review
+- Make the PR as small as possible: it's much easier to review five 200 lines PR than one 1000 lines PR
+- If a PR is known to create conflict with other active PRs, try to coordinate, and link to each other
+  - if appropriate explain desired review/merge order 
+- For complex changes that will require multiple PR, create an issue to keep track of what has been done and what's left to do.
+- Is the PR focused on a single bugfix or feature?
+  - consider moving self contained changes to separate PRs 
+
+Public Interfaces
+-----------------
+
+- are there comprehensive tests
```

Comment:
- How does one know if the proposed tests are comprehensive?
- Can we firm up the concept, such that we give PR author a clear, actionable set of ideas on the minimum set of tests needed for a focused PR to pass muster?

Reply:
Don't think so, not without writing overly much and then having to constantly say: yeah ok this aspect was not covered in our detailed list because its impossible to make a detailed list since what comprehensive means may depend on the functionality tested.

Reply:
Maybe split the difference and say something along the lines of "at least test coverage for the most obvious modes of failure" ...?

Reply:
No that is not what I mean: we need to test all ways something could go wrong.